### ISSUE TITLE
Make a PyPI release

### ISSUE THREAD

**Issue by carljm** [fwenzel/django-sha2#11]
Is there a reason this project isn't released on PyPI?

**Comment by fwenzel** [comment]
Nope, no ill intent. Just haven't gotten around to it yet, and since it'd be my first release on PyPI, I'll first have to figure out how it works. Thanks for filing this!

**Comment by andymckay** [comment]
Bump, any chance we could get this release.

**Comment by fwenzel** [comment]
@andymckay if you're on a roll making PyPI releases today, I'm happy to give you access to this repo and let you release it.

**Comment by andymckay** [comment]
Sounds good.

**Comment by jsocol** [comment]
@fwenzel @andymckay Ping. How about a 0.5 on PyPI?
